packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -21,12 +21,15 @@ const WalletsBanner = makeLazyLoader(
 
 const MainTitleBar = () => {
     const { traders_hub, client } = useStore();
+    const { is_landing_company_loaded, is_switching } = client;
     const { state: wallet_migration_state } = useWalletMigration();
     const { selected_region, handleTabItemClick, toggleRegulatorsCompareModal, content_flag } = traders_hub;
-    const { is_landing_company_loaded, is_switching } = client;
+    const { is_next_wallet_enabled } = useFeatureFlags();
+
     const is_low_risk_cr_real_account =
         content_flag === ContentFlag.LOW_RISK_CR_NON_EU || content_flag === ContentFlag.LOW_RISK_CR_EU;
-    const show_wallets_banner = wallet_migration_state && wallet_migration_state !== 'ineligible';
+    const show_wallets_banner =
+        is_next_wallet_enabled && wallet_migration_state && wallet_migration_state !== 'ineligible';
```

Comment:
I think for migration, we need to use different flag. Either something from backend or something which we can set on frontend per specific country etc.

Reply:
We will be removing the `is_next_wallet_enabled` completely soon; for now this flag's function essentially being transformed into `is_wallet_migration_allowed` in this PR